docs_src/examples/app-service-examples/camera-management/Ch-CameraManagement.md

```diff
@@ -0,0 +1,426 @@
+Use the Camera Management Example application service to auto discover and connect to nearby ONVIF and USB based cameras. This application will also control cameras via commands, create inference pipelines for the camera video streams and publish inference results to MQTT broker.
+
+This app uses [EdgeX compose][edgex-compose], [Edgex Onvif Camera device service][device-onvif-camera], 
+[Edgex USB Camera device service][device-usb-camera], [Edgex MQTT device service][device-mqtt] and [Edge Video Analytics Microservice][evam].
+
+
+## Install Dependencies
+
+### Environment
+This example has been tested with a relatively modern Linux environment - Ubuntu 20.04 and later
+
+### Install Docker
+Install Docker from the official repository as documented on the [Docker](https://docs.docker.com/engine/install/ubuntu/) site.
+
+### Configure Docker
+To enable running Docker commands without the preface of sudo, add the user to the Docker group.
+
+!!! warning
+    The docker group grants root-level privileges to the user. For details on how this impacts security in your system, see [Docker Daemon Attack Surface](https://docs.docker.com/engine/security/#docker-daemon-attack-surface).
+
+1. Create Docker group:
+    ```bash
+    sudo groupadd docker
+    ```
+
+    !!! note
+        If the group already exists, `groupadd` outputs a message: **groupadd: group `docker` already exists**. This is OK.
+
+2. Add User to group:
+   ```bash
+   sudo usermod -aG docker $USER
+   ```
+
+3. Restart your computer for the changes to take effect.
+
+### Verify Docker
+To verify the Docker installation, run `hello-world`:
+
+```bash
+docker run hello-world
+```
+A **Hello from Docker!** greeting indicates successful installation.
+
+```bash
+Unable to find image 'hello-world:latest' locally
+latest: Pulling from library/hello-world
+2db29710123e: Pull complete 
+Digest: sha256:10d7d58d5ebd2a652f4d93fdd86da8f265f5318c6a73cc5b6a9798ff6d2b2e67
+Status: Downloaded newer image for hello-world:latest
+
+Hello from Docker!
+This message shows that your installation appears to be working correctly.
+...
+```
+
+### Install Docker Compose
+Install Docker Compose from the official repository as documented on the [Docker Compose](https://docs.docker.com/compose/install/linux/#install-using-the-repository) site.
+
+### Install Golang
+Install Golang from the official [Golang](https://go.dev/doc/install) website.
+
+### Install Tools
+Install build tools:
+
+```bash
+sudo apt install build-essential
+```
+
+## Steps for running this example:
+
+### 1. Start the EdgeX Core Services and Device Services.
+
+1. Clone `edgex-compose` from github.com.
+    ```shell 
+    git clone https://github.com/edgexfoundry/edgex-compose.git
+    ```  
+
+1. Navigate to the `edgex-compose` directory:
+    ```bash
+    cd edgex-compose
+    ```
+
+1. Checkout the latest compatible release branch
+
+    !!! note
+        The `{{version}}` branch is the latest stable branch at the time of this writing.
```

Comment:
Should we have this warning for the WIP branch too? I think we should not have or modify it.

Reply:
IMO, we don't need this note since the user has been told to use the latest release..

Reply:
I think that's for the best. Especially if all of the main Readmes contain a similar warning, it's probably best to stay away from making specific references about the status of any release